docs/technical_reference/unit_models/coag_floc_model.rst

```diff
@@ -0,0 +1,160 @@
+Coagulation-Flocculation
+========================
+
+Introduction
+------------
+
+Coagulation-Flocculation is a water treatment process  designed to remove total suspended
+solids (TSS, primarily as natural organic matter) from solution by converting those small,
+suspended particles into larger 'floccs' (or 'Sludge') that can then be separated by sedimentation
+and/or filtration (or other separation processes) later in the treatment train. This
+is accomplished in a 2-stage process:
+
+Stage 1) Coagulation (where chemical coagulants and other aids are rapidly mixed into solution)
+Stage 2) Flocculation (where the 'floccs' are formed through slow, gentle mixing/agitation)
+
+In this implementation of the model, the user MUST provide a measured final Turbidity (in NTU) made
+during a Jar Test for the given water source. This measurement is then used to estimate how much
+TSS would be removed during the Coagulation-Flocculation process. User's may also

```

Comment:
I see six instances in `coag_floc_model.rst` where "user's" should be changed to "users". Other than that, everything looks fine to me.